third_party/move/move-bytecode-verifier/bytecode-verifier-tests/src/unit_tests/vec_pack_tests.rs

```diff
@@ -67,5 +67,5 @@ fn test_vec_pack() {
         &m,
     )
     .unwrap_err();
-    assert_eq!(res.major_status(), StatusCode::VALUE_STACK_PUSH_OVERFLOW);
```

Comment:
Note: we can fail with a different status code, but this should be protected by feature flag.